modules/schema/src/main/java/org/apache/ignite/internal/schema/mapping/ColumnMapperImpl.java

```diff
@@ -17,34 +17,56 @@
 
 package org.apache.ignite.internal.schema.mapping;
 
+import org.apache.ignite.internal.schema.Column;
+import org.apache.ignite.internal.schema.SchemaDescriptor;
+import org.jetbrains.annotations.NotNull;
+
 /**
  * Column mapper implementation.
  */
 class ColumnMapperImpl implements ColumnMapper, ColumnMapperBuilder {
```

Comment:
I would prefer either split this onto two different classes or merge ColumnMapper and ColumnMapperBuilder interfaces. The builder pattern is used usually  to create an object, not to change its state.